ui/apps/platform/src/hooks/useURLSort.ts

```diff
@@ -1,33 +1,22 @@
-import { useRef, useState } from 'react';
-import useDeepCompareEffect from 'use-deep-compare-effect';
+import { useRef } from 'react';
+import findIndex from 'lodash/findIndex';
+import intersection from 'lodash/intersection';
+import isEqual from 'lodash/isEqual';
 
-import useURLParameter, { HistoryAction } from 'hooks/useURLParameter';
-import { SortAggregate, SortDirection, SortOption, ThProps } from 'types/table';
-import { ApiSortOption } from 'types/search';
-import { isParsedQs } from 'utils/queryStringUtils';
+import useURLParameter, { HistoryAction, QueryValue } from 'hooks/useURLParameter';
+import { SortAggregate, SortOption, ThProps, isSortOption } from 'types/table';
+import { ApiSortOption, ApiSortOptionSingle } from 'types/search';
 
 export type GetSortParams = (
-    field: string,
-    aggregateBy?: SortAggregate
+    columnName: string,
+    fieldOptions?: SortAggregate | { field: string; aggregateBy?: SortAggregate }[]
```

Comment:
Parameter renaming to more closely match the intended purpose.

Since `field` is only sent to the API when using the single sort option, it was changed to the more general `columnName` and documented that it will be used as the "field" during API requests.